src/Components/EditAdditionalRepositoriesTable/EditAdditionalRepositoriesTable.js

```diff
@@ -277,19 +296,31 @@ const EditAdditionalRepositoriesTable = (props) => {
   const searchedRepos = filterReposBySearch(sortedRepositories, searchValue);
   const paginatedRepos = getPage(searchedRepos);
 
-  const isRepoSelectable = (repo) => repo.repositoryName !== 'a';
+  React.useEffect(() => {
+    setAdditionalRepos(selectedRepoLabel);
+  }, [selectedRepoLabel]);
+
+  const isRepoSelectable = (repo) => repo.repositoryLabel !== 'a';
 
   const setRepoSelected = (repo, isSelecting = true) =>
-    setSelectedRepoNames((prevSelected) => {
-      const otherSelectedRepoNames = prevSelected.filter(
-        (r) => r !== repo.repositoryName
+    setSelectedRepoLabel((prevSelected) => {
+      const otherSelectedRepoLabel = prevSelected.filter(
+        (r) => r !== repo.repositoryLabel
       );
       return isSelecting && isRepoSelectable(repo)
-        ? [...otherSelectedRepoNames, repo.repositoryName]
-        : otherSelectedRepoNames;
+        ? [...otherSelectedRepoLabel, repo.repositoryLabel]
+        : otherSelectedRepoLabel;
     });
   const isRepoSelected = (repositories) =>
-    selectedRepoNames.includes(repositories?.repositoryName);
+    selectedRepoLabel.includes(repositories?.repositoryLabel);
+
+  const saveDisabled = () => {
+    if (selectedRepoLabel.length === 0) {
+      return true;
+    } else {
+      return false;
+    }
+  };
```

Comment:
This can be more concise
```suggestion
  const saveDisabled = selectedRepoLabel.length === 0;
```

Reply:
I don't think it needs to be a function, but if it does then
```suggestion
  const saveDisabled = () => selectedRepoLabel.length === 0;
```
is still an improvement

Reply:
I would also rename it to `isSaveDisabled` so that it matches the pattern of `isDisabled` for the patternfly component